src/main/java/xyz/nucleoid/spleef/game/SpleefTimerBar.java

```diff
@@ -42,11 +40,11 @@ private Text getDroppingText(long ticksUntilDrop) {
         long seconds = secondsUntilDrop % 60;
         var time = String.format("%02d:%02d", minutes, seconds);
 
-        return getBarTitle(new TranslatableText("text.spleef.bar.dropping", time));
+        return getBarTitle(new TranslatableText("text.spleef.bar.dropping", time), "GREEN");
     }
 
-    private static Text getBarTitle(Text customText) {
+    private static Text getBarTitle(Text customText, String color ) {
         var gameName = new TranslatableText("gameType.spleef.spleef").formatted(Formatting.BOLD);
-        return new LiteralText("").append(gameName).append(" - ").append(customText).formatted(Formatting.GREEN);
+        return new LiteralText("").append(gameName).append(" - ").append(customText).formatted(Formatting.valueOf(color));
```

Comment:
You can use the enum as the type directly rather than relying on `valueOf` here.